exercises/setting-colours.md

```diff
@@ -0,0 +1,33 @@
+
+## Setting colours
+
+
+We've seen how to change the text content of html elements, but we can also change the colour of elements.  Different kinds of **Values blocks** provide ways of setting texts, numbers and colours.
+
+Let's start with an html list of our favourite fruit in the "html" section.
+
+1.  each html list item (`<li>`) has a different id attribute so that we can refer to it using a css selector
+
+For example: `<ul>
+        <li id="banana">Banana</li>  
+        <li id="apple">Apple</li>  
+        <li id="strawberry">Strawberry</li>  
+        </ul>`
+        
+2.  Click "run" to see the rendered output
+
+Let's change the banana element's colour to yellow
+    
+3.  Add an "at the start" block
+
+4.  Inside this block, add a "find the element using css selector" block using the id selector for the <li> element for Banana (`#banana`)
```

Comment:
Should I start removing reference to `css selector` now in favour of `get element by id`

Reply:
in index.html, they are currently removed. Maybe we need a first commit that pulls everything out into .md files and has a build phase that assembles them all back together?

Reply:
Yes definitely, that seems sensible.
Have you got time to discuss tomorrow ?

Reply:
Yep. Some time between 4 and 6pm? I've started looking at server side/rendering of parsing of .md files and gotten a bit stuck

Reply:
Yes, that works well for me